garden_ai/utils/_meta.py

```diff
@@ -0,0 +1,144 @@
+import inspect
+import linecache
+import textwrap
+
+
+def redef_in_main(cls):
+    """Helper: redefine a class in __main__, e.g. garden_ai._Model -> __main__._Model.
```

Comment:
nit: This isn't _just_ for classes, right? Seeing as we apply it to `garden_ai_compose_functions`.

Reply:
oh yeah good catch, that comment / the variable `cls` is outdated from when we were only using it for a class definition